integration/query_compat_test.go

```diff
@@ -80,9 +80,17 @@ func testQueryCompat(t *testing.T, testCases map[string]queryCompatTestCase) {
 				t.Run(targetCollection.Name(), func(t *testing.T) {
 					t.Helper()
 
-					// Run `explain` on `targetCollection` only, check response's `pushdown` with tc.resultPushdown
-					// https://github.com/FerretDB/FerretDB/issues/1279
-					_ = tc.resultPushdown
+					explainQuery := bson.D{{"explain", bson.D{
+						{"find", targetCollection.Name()},
+						{"filter", filter},
+						{"sort", opts.Sort},
+						{"projection", opts.Projection},
+					}}}
+
+					var explainRes bson.D
+					require.NoError(t, targetCollection.Database().RunCommand(ctx, explainQuery).Decode(&explainRes))
+
+					assert.Equal(t, tc.resultPushdown, explainRes.Map()["pushdown"])
```

Comment:
do we want to check the result found by push down query?

Reply:
I believe it's not in the scope of this issue. We need a simple check to have any way to test the future pushdowns

Reply:
:+1: